src/commands/push-image.yml

```diff
@@ -18,11 +18,19 @@ parameters:
     type: string
     default: "latest"
 
+  return-sha-var:
+    description: Variable you want set with the container sha (default = SHA256)
+    type: string
+    default: SHA256
+
 steps:
   - run:
       name: Push image to Amazon ECR
       command: |
         IFS="," read -ra DOCKER_TAGS \<<< "<< parameters.tag >>"
         for tag in "${DOCKER_TAGS[@]}"; do
-          docker push $<<parameters.account-url>>/<<parameters.repo>>:${tag}
+          PUSH=$(docker push $<<parameters.account-url>>/<<parameters.repo>>:${tag} | base64 --wrap=0)
+          echo $PUSH | base64 --decode
         done
+        SHA=$(echo $PUSH | base64 --decode | grep sha256 | cut -d " " -f3)
```

Comment:
Can I make one pedantic request (technically two)? 

Can we prefix these environment variables? Only because I feel they are generic enough that users may run into some kind of conflicting error.

For the internal env vars `PUSH` and `SHA`, how about something similar to: `ECR_IMG_PUSH` and `ECR_IMG_SHA`.  I think we could consider prefixing the default as well: `ECR_IMG_SHA256`

I think that may just save us in the future, happy to merge after 👍 thank you! 🙏

Reply:
How did you know my normal variables for use were ECR_IMG_SHA256 and ECR_IMG_PUSH?! uncanny!

I debated forking bash for scoped variables instead :P